prime-router/settings/organizations-local.yml

```diff
@@ -143,7 +143,7 @@
           redox_source_id: "d89d4057-930f-4c5b-bb39-8cddb326e928"
           redox_destination_id: "62d62d52-3771-4151-aff4-4a3d420a8b7a"
           redox_destination_name: "Colorado Dept of Public Health (CDPHE)"
-        nameFormat: "STANDARD"
+        nameFormat: "standard"
```

Comment:
`nameFormat` is now a string and lower case and matches the names above in the file name templates metadata.